Composer/packages/client/src/components/QnA/CreateQnAModal.tsx

```diff
@@ -535,6 +444,9 @@ export const CreateQnAModal: React.FC<CreateQnAModalProps> = (props) => {
           </div>
         </div>
         <DialogFooter>
+          <div style={{ float: 'left' }}>
+            <PersonalCard />
```

Comment:
Should probably call this "PersonaCard" as "Persona" is the underlying component name.